src/app/screens/App/screens/Home/components/Book/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { BOOKCOVERALTTEXT } from '../../strings';
+import './styles.css';
+import defaultBookImage from '../../../../../../../assets/ASSETS/default_book.svg';
+
+export default function Book(props) {
+  const has_image = props.book.image_url;
+  const image_url = props.book.image_url || defaultBookImage;
+
+  return (
+    <div key={props.book.id} className="book">
+      <div
+        className={
+          'book-image-container ' + (has_image ? '' : 'backgroundgray')
+        }
+      >
+        <img
+          src={image_url}
+          className={has_image ? 'book-image' : 'default-image'}
+          alt={BOOKCOVERALTTEXT}
+        />
+      </div>
+      <span className="book-title">{props.book.title}</span>
+      <span className="book-author">{props.book.author}</span>
+    </div>
+  );
+}
```

Comment:
start to add proptypes from now.

import `prop-types` package

and then do:

class  TheName ... {
   ...
}

TheName.proptypes = {
  title: Proptypes.string,
  id: Proptypes.number
  ....
}

So React will check the type of received props and shot a warning if you are receiven a incorrect data type

for array:
`Proptypes.arrayOf(Proptype.any)`
for objects:
`Proptypes.shape({ [keys]: [values] })`

Reply:
do this for all components